### DISCUSSION TITLE
A separate options panel for interactive plot

### DISCUSSION THREAD

**Discussion by sergei9838** [marcomusy/vedo#1124] [Q&A] [answered]
Hi, Marco,
Is it possible to have a separate panel with sliders, switches, maybe, typed in options for an interactive plot? When, say, 6 sliders and two buttons are put in one renderer window together with help text, it looks... overwhelming. I mean having something like this drawn by the main renderer:
[https://panel.holoviz.org/gallery/vtk_interactive.html](https://panel.holoviz.org/gallery/vtk_interactive.html)
<img width="362" alt="Screenshot 2024-05-21 at 09 19 04" src="https://github.com/marcomusy/vedo/assets/105316522/e4611024-e2de-45d0-a56f-53dca895f014">

A quick fix could be to have a separate renderer window with all sliders and buttons put there, but maybe there is a better solution?
Thank you!

**Comment by marcomusy** [comment, marked as answer]
Yes - window sliders and buttons are designed for very simple user interfaces (which in my experience covers most of the practical cases).
For more complex GUI you can look at Qt which is supported and magic-gui which is based on it. Examples are:

```
examples/other/magic-class1.py
examples/other/qt_*.py
```
